rosjava/src/main/java/org/ros/internal/node/DefaultNode.java

```diff
@@ -416,7 +416,7 @@ public void shutdown() {
     // NOTE(damonkohler): We don't want to raise potentially spurious
     // exceptions during shutdown that would interrupt the process. This is
     // simply best effort cleanup.
-    slaveServer.shutdown();
+    //slaveServer.shutdown();
```

Comment:
Please keep this line and remove the call below: https://github.com/rjcausarano/rosjava_core/pull/1/files#diff-0773456238fe32ede712ed928108c550R440.